data/monster/learnset/learnset_ptrs.s

```diff
@@ -0,0 +1,840 @@
+.4byte 0
```

Comment:
I would prefer if this file was also generated. So if someone wanted to add another pokemon, it would be simple.



Was able to get match the output with the following commands below. We could add a rule that just executes this.

```bash
echo '.4byte 0' >> learnset_ptrs.s
echo '.4byte 0' >> learnset_ptrs.s
cat learnset_data.inc | grep .global >> learnset_ptrs.s
sed -i 's/global/4byte/g' learnset_ptrs.s
```

Reply:
Added this to `data_monster.mk`.